src/NHibernate.Config.Templates/SapSQLAnywhere.cfg.xml

```diff
@@ -15,7 +15,7 @@ for your own use before compiling tests in Visual Studio.
 		<property name="connection.connection_string">
 			UID=DBA;PWD=sql;Server=localhost;DBN=nhibernate;DBF=c:\nhibernate.db;ASTOP=No;Enlist=false;
 		</property>
-		<property name="dialect">NHibernate.Dialect.SybaseSQLAnywhere12Dialect</property>
+		<property name="dialect">NHibernate.Dialect.SapSQLAnywhere17Dialect</property>
```

Comment:
We cannot test it locally if staying on the old dialect. That is only the template, so, the trouble can be fixed locally. But why not fixing that here?